### ISSUE TITLE
Upload speed test hangs

### ISSUE THREAD

**Issue by vvmnnnkv** [OpenMined/syft.js#150]
## Description
Separated from #148:

The upload speed test takes extremely long and blocks the whole machine. I had to manually kill my chrome process to exit the syft.js tab. This is weird because the 64MB buffer you created should not take too long to send to a localhost server. I solved this by just returning Promise.resolve(1000) in the getUploadSpeed function.

@erksch Could you provide more details? OS and Browser version?

## How to Reproduce
2. Click on '...'
3. Scroll down to '...'

## System Information
 - OS Version: [e.g. 22]
 - Language Version: [e.g. Python 3.7, Node 10.18.1]
 - Package Manager Version: [e.g. Conda 4.6.1, NPM 6.14.1]

**Comment by cereallarceny** [comment]
@erksch could you add context here?

**Comment by erksch** [comment]
Yes! Sorry, I am a little bit occupied at the moment, I hope to try it again and give more details in the next days.
